CHANGELOG.md

```diff
@@ -1,13 +0,0 @@
-# Change Log
```

Comment:
The new release-process requires new releases to be created in github. The pipeline will then automatically publish the extension. Changes in new versions are documented in the github release tags.

This workflow makes the CHANGELOG.md file obsolete. 

The downside of the new process is that the Changelog will not be available on the visuals studio marketplace extension page.